tests/__sf_section_conda.test.fish

```diff
@@ -0,0 +1,80 @@
+source $DIRNAME/spacefish_test_setup.fish
+source $DIRNAME/mock.fish
+set -l LOCAL_CONDA_VERSION 4.5.11
+
+function setup
+    spacefish_test_setup
+    mock conda 0 "echo \"4.5.11\""
```

Comment:
Is this mock exactly what is echoed by conda?
I thought there'd be more since there's the `cut` logic.

Reply:
aaah!